docker/generic/start_proxy.py

```diff
@@ -506,6 +513,9 @@ def enforce_conflict_args(args):
     if args.backend_dns_lookup_family and args.backend_dns_lookup_family not in {"auto", "v4only", "v6only"}:
         return "Flag --backend_dns_lookup_family must be 'auto', 'v4only' or 'v6only'."
 
+    if args.ssl_port and args.ssl_server_path:
+        return "Flag --ssl_port is going to be deprecated, please use --ssl_server_path only."
```

Comment:
Nit: Make it clear that `--listener_port` / `--http_port` will service SSL traffic in this case.

Reply:
it is stated on --ssl_server_path.